interfaces/cython/cantera/units.pyx

```diff
@@ -18,8 +19,38 @@ cdef class Units:
         if name:
             self.units = CxxUnits(stringify(name), True)
 
-    def __repr__(self):
-        return f"<Units({pystr(self.units.str())}) at {id(self):0x}>"
+    def __repr__(self) -> str:
+        return f"<Units({str(self)}) at {id(self):0x}>"
+
+    def __str__(self) -> str:
+        return pystr(self.units.str())
+
+    def dimension(self, primary: str) -> float:
+        """The dimension of the given unit component.
+
+        .. versionadded:: 3.0
+
+        :param: primary, a string with the desired unit component. One of ``"mass"``,
```

Comment:
```suggestion
        :param primary:
            A string with the desired unit component. One of ``"mass"``,
```
This is mostly a question: is this `:param: primary ...`, or `:param primary: ...`? Most other instances use the latter formatting ... (there are only two instances of the former, which should probably be taken care of)

Reply:
Here are the only two instance of the former in all of Cantera: https://github.com/Cantera/cantera/blob/790b507113d79c4901e319ed8837e5e7cbaaca1f/interfaces/cython/cantera/thermo.pyx#L735-L740
(and it is likely incorrect usage) ... it may be be worth taking care of.